endtoend_tests/forseti/scanner/enabled_apis_scanner_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Enabled APIs Scanner test"""
+
+import pytest
+import re
+from sqlalchemy.sql import text
+
+
+class TestEnabledApisScannerTest:
```

Comment:
I feel the same test can be used for a bunch of scans if we parameterize the violation_type, violation_rule_name and so on.
In this way, we only need to write one test and just provide an array of parameters for different rules.